### DISCUSSION TITLE
Spaces in name field binding Handlebars

### DISCUSSION THREAD

**Discussion by Keksik-tyan** [Budibase/budibase#5956] [Help] [answered]
Hi!

I have a field named "Documents is deleted". I want to check it in a condition in automation. However, when saving, the error is "Current Handlebars syntax is invalid, please check the guide here for more details".

![image](https://user-images.githubusercontent.com/42870688/169031392-e7b06f35-bc24-4ef5-8978-75a8cecb665a.png)

Can't use spaces with field names when automating?

**Comment by melohagan** [comment, marked as answer]
Hi @Keksik-tyan 

I had noticed that myself, and raised the bug here: https://github.com/Budibase/budibase/issues/5715

Thanks for highlighting nonetheless!

**Reply by melohagan** [reply]
As Peter pointed out though, you can use square brackets like so:

`{{ trigger.row[Documents is deleted] }}`

**Reply by Keksik-tyan** [reply]
@melohagan, Thanks for your help :)
